common_code/poisson_operator.h

```diff
@@ -114,9 +114,8 @@ namespace Poisson
           all_indices_uniform.resize(Utilities::pow(3, dim) * data->n_cell_batches(), 1);
         }
 
-      FE_Nothing<dim> dummy_fe;
-      FEValues<dim>   fe_values(*data_->get_mapping_info().mapping,
-                              dummy_fe,
+      FEValues<dim> fe_values(*data_->get_mapping_info().mapping,
+                              data_->get_dof_handler().get_fe(),
```

Comment:
This should not have had happened! Let me check what went wrong on dealii/master.

Reply:
I think the error is fine. It was introduced by the geometry commit in #10 I believe.

Reply:
Oh, right! I interpreted the error differently. I thought the new constructor of `FE_Nothing` makes problems...